CliClient/tests/md_to_html/video.md

```diff
@@ -0,0 +1,12 @@
+[](http://www.youtube.com/watch?v=iwGFalTRHDA)
+[](https://www.youtube.com/watch?v=iwGFalTRHDA) 
+[](https://www.youtube.com/watch?v=iwGFalTRHDA&feature=related)
+[](http://www.youtube.com/embed/watch?feature=player_embedded&v=iwGFalTRHDA)
+[](http://www.youtube.com/embed/watch?v=iwGFalTRHDA)
+[](http://www.youtube.com/watch?feature=player_embedded&v=iwGFalTRHDA)
+[](http://www.youtube.com/watch?v=iwGFalTRHDA)
+[](http://www.youtube.com/attribution_link?u=/watch?v=aGmiw_rrNxk&feature=sharea=9QlmP1yvjcllp0h3l0NwuA)
+[](http://www.youtube.com/attribution_link?a=fF1CWYwxCQ4&u=/watch?v=qYr8opTPSaQ&feature=em-uploademail)
+[](http://www.youtube.com/attribution_link?a=fF1CWYwxCQ4&feature=em-uploademail&u=/watch?v=qYr8opTPSaQ)
+https://www.youtube.com/watch?v=SADub7W22Zg
+[google](https://www.google.com/)
```

Comment:
Thanks for the detailed tests. It also need to work with these cases:

- `[some title here](http://www.youtube.com/watch?v=iwGFalTRHDA)` => should create a video player and display a link "some title here" below the video, which opens the video on YouTube.
- `http://www.youtube.com/watch?v=iwGFalTRHDA` => should create a video player
- `https://www.a.com/watch?v=SADub7W22Zg` => should **not** create a video player (it currently does!)

Reply:
![image](https://user-images.githubusercontent.com/54576074/79128545-51a2b800-7dc1-11ea-89e5-a0f32193b289.png)